bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/EventSubType.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api.data;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * This enum describes sub events in relation to a given event
+ * according to API documentation
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public enum EventSubType {
+    MISSING_SD(EventType.SD, 1), // Missing SD Card
+    SD_INSERTED(EventType.SD, 2), // SD Card inserted
+    SD_FORMATED(EventType.SD, 3), // SD Card formated
```

Comment:
formatted

Reply:
Maybe also rename to MISSING_SD_CARD, SD_CARD_INSERTED and SD_CARD_FORMATTED and get rid of the comments?

Reply:
I agree

Reply:
Done

Reply:
Please formatted with 2 t

Reply:
Done